@ndlib/gatsby-theme-marble/src/assets/i18n/json/common.en.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This file will be left blank, so that users can override the values in the site folder.